whitepapers/ingest/Graph.md

```diff
@@ -0,0 +1,29 @@
+# MutableGraph [NOT IMPLEMENTED]
```

Comment:
In the context of the other libraries I was looking at, networkX was the slowest and doesn't implement graph filtering. I think my main issue with the library was the lack of graph filtering.  If we choose to use it we would need to augment the functionality a little bit but it should probably be reconsidered for the base of our graph interface. That being said, the additions would look a lot like what Abram's already done here -- the code would just need to be refactored to use networkX.

See https://graph-tool.skewed.de/performance
> NetworkX, on the other hand, comes at a distant third with running times in the order of 20 to 170 times slower than graph-tool. This is mostly due to its pure Python implementation, which is known in general to be substantially slower than C/C++ (see here, here and here for further comparisons).

[My Evaluation](https://gist.github.com/laurenbarker/97d28ff75f15a4b42a2646d8dd957837)